packages/stream/common/types.ts

```diff
@@ -25,6 +25,7 @@ export interface IStreamConfig {
   tokenId: string;
```

Comment:
Nit:
Code above - 
![image](https://user-images.githubusercontent.com/125874670/221863372-3deff66d-85d3-455c-be4e-4d4713df2ee9.png)
Why do we need this comment, and should we have JSDocs in these interfaces?
I don't know what partner is